apps/docs/src/app/core/component-docs/facets/facet-examples/facet-group-example.component.html

```diff
@@ -0,0 +1,68 @@
+<fd-facet-group ariaLabel="Facet Group">
+    <fd-facet type="form" facetTitle="Technical Data" id="facet1">
+        <fd-facet-content>
+            <label class="fd-form-label" for="form-value-10">Base unit:</label>
+            <div class="fd-text" id="form-value-10">Each</div>
+        </fd-facet-content>
+        <fd-facet-content>
+            <label class="fd-form-label" for="form-value-11">Length:</label>
+            <div class="fd-text" id="form-value-11">23.24 Centimeter</div>
```

Comment:
I know it's only example, but could you use `fd-text` and `fd-form-label` directives? Instead of classes

Reply:
On adding `fd-form-label` directive, it adds other styles by default that give extra padding between form and label:
![image](https://user-images.githubusercontent.com/53509521/112299062-71bf4480-8cbd-11eb-8f48-a1e4069193d6.png)
Fiori3 recommends only 0.5rem spacing between form and label for a Form facet so adding the directive doesn't comply.

Reply:
@JKMarkowski only replaced `fd-text` and not `fd-form-label` because of the above issue. Please check.